rclpy/rclpy/timer.py

```diff
@@ -14,19 +14,34 @@
 
 import threading
 
+from typing import Callable
+from typing import Optional
+
+from rclpy.callback_groups import CallbackGroup
+from rclpy.clock import Clock
+from rclpy.context import Context
 from rclpy.exceptions import InvalidHandle, ROSInterruptException
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
 from rclpy.utilities import get_default_context
 
 
 class Timer:
 
-    def __init__(self, callback, callback_group, timer_period_ns, clock, *, context=None):
+    def __init__(
+        self,
+        callback: Callable,
+        callback_group: CallbackGroup,
+        timer_period_ns: int,
+        clock: Clock,
+        *,
```

Comment:
Should not it be `*args`? And should not it be positioned after all the other parameters? If so, you will not need to combine named and unnamed parameters [here](https://github.com/ros2/rclpy/pull/1138/files#diff-610fc172e919c8a045bd8f7aec9732a4830668c8bace69bc95132bd07aedbf31R1699)

Reply:
No, there should be no changes here.  We want this to be a kwargs, just like `context`.